zeppelin-web/src/app/interpreter/interpreter.html

```diff
@@ -192,6 +192,48 @@ <h3 class="interpreter-title">{{setting.name}}
       </div>
 
 
+      <div class="col-md-12">
+        <div class="checkbox">
+          <span class="input-group" style="line-height:30px;">
+            <label><input type="checkbox" style="width:18px !important" id="idShowPermission" ng-click="togglePermissions()" ng-model="setting.option.setPermission" ng-disabled="!valueform.$visible"/>
+            Set permission </label>
+          </span>
+        </div>
+      </div>
+
+      <div class="col-md-12">
+        <!-- permissions -->
+        <div ng-show="setting.option.setPermission" class="permissionsForm">
+          <div>
+            <p>
+              Enter comma separated interpreters in the fields. <br />
```

Comment:
Don't we need to type comma separated `users` list not `interpreters` in this field?